graylog2-web-interface/src/views/components/Query.jsx

```diff
@@ -56,7 +60,45 @@ const _renderWidgetGrid = (widgetDefs, widgetMapping, results, positions, queryI
   );
 };
 
+const EmptyDashboardInfo = () => (
+  <Row className="content" style={{ marginRight: 0, marginLeft: 0 }}>
+    <Col md={12}>
+      <Jumbotron style={{ marginBottom: 0 }}>
+        <h2>
+          <IfDashboard>
+            This dashboard has no widgets yet
+          </IfDashboard>
+          <IfSearch>
+            There are no widgets defined to visualize the search result
+          </IfSearch>
+        </h2>
+        <br />
+        <p>
+          Create a new widget by selecting a widget type in the left sidebar section &quot;Create&quot;.<br />
+        </p>
+        <p>
+          A few tips for creating searches and dashboards
+        </p>
+        <ul>
+          <li><p>1. Start with a <b>question</b> you want to answer. Define the problem you want to solve.</p></li>
+          <li><p>2. <b>Limit</b> the data to only the data points you want to see.</p></li>
+          <li><p>3. <b>Visualize</b> the data. Does it answer your question?</p></li>
+          <li><p>4. <b>Share</b> the view with your colleagues. Prepare it for <b>reuse</b> by using parameters (contained in <a href="https://www.graylog.org/graylog-enterprise-edition" target="_blank" rel="noopener noreferrer">Graylog Enterprise</a>).</p></li>
+        </ul>
```

Comment:
I had to write the list this way (for now), because of the described Jumbotron behaviour. (see other comments)

Reply:
Good point! We can replace `view` with `dashboard` here and render the fourth point conditionally using `IfDashboard`.